app/src/main/java/com/nervousfish/nervousfish/activities/ContactReceivedHelper.java

```diff
@@ -41,17 +45,22 @@
      * @param contact  The newly received {@link Contact}
      */
     static void newContactReceived(final IDatabase database, final Activity activity, final Contact contact) {
+        if (contact == null) {
```

Comment:
Well, since all communication is within our application that should never happen 🤔 The reason I did it was so that you don't have to do this check manually [in the MainActivity](https://github.com/ericcornelissen/NervousFish/pull/297/files#diff-149538cc1f1f5e4370b6ae93d6489330R108), and potentially other activities. But if you prefer to have it changed, I'm OK with that 😄